tests/Integration/Behaviour/Features/Context/CustomerFeatureContext.php

```diff
@@ -102,6 +103,40 @@ public function setCurrentCustomer($customerName)
         Context::getContext()->updateCustomer($this->customers[$customerName]);
     }
 
+    /**
+     * @Given private note is not set about customer :reference
+     */
+    public function assertPrivateNoteIsNotSetAboutCustomer($reference)
+    {
+        /** @var Customer $customer */
+        $customer = SharedStorage::getStorage()->get($reference);
+
+        if ($customer->note) {
+            throw new RuntimeException(sprintf(
+                'It was expected that customer "%s" should not have private note.',
+                $reference
+            ));
+        }
+    }
+
+    /**
+     * @Then customer :reference private note should be :privateNote
```

Comment:
```suggestion
     * @Then customer :reference private note content should be :privateNote
```
Maybe we should use a [Bheat pystring](https://docs.behat.org/en/v2.5/guides/1.gherkin.html#pystrings) here in order to allow more complex notes to be specified ?

Reply:
Not sure, note is a simple string. :thinking:

Reply:
> Not sure, note is a simple string.
Then we can use a single-column-single-row Behat `TableNode`

```
* @Then customer :reference private note content should be :
| Hey this is a not I wrote for Sarunas ! |
```

> Regarding your wording, do we really need to point out "content"? There is nothing else (header, footer & etc) about it but note. 🤔

Maybe it was overthinking 😺 thanks for pointing this out